xrdp/xrdp_egfx.h

```diff
@@ -0,0 +1,173 @@
+/**
+ * xrdp: A Remote Desktop Protocol server.
+ *
+ * Copyright (C) Jay Sorg 2019
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * MS-RDPEGFX
+ */
+
+#ifndef _XRDP_EGFX_H
+#define _XRDP_EGFX_H
+
+#define XR_RDPGFX_CAPVERSION_8      0x00080004
+#define XR_RDPGFX_CAPVERSION_81     0x00080105
+#define XR_RDPGFX_CAPVERSION_10     0x000A0002
+#define XR_RDPGFX_CAPVERSION_101    0x000A0100
+#define XR_RDPGFX_CAPVERSION_102    0x000A0200
+#define XR_RDPGFX_CAPVERSION_103    0x000A0301
+#define XR_RDPGFX_CAPVERSION_104    0x000A0400
+#define XR_RDPGFX_CAPVERSION_105    0x000A0502
+#define XR_RDPGFX_CAPVERSION_106    0x000A0600 /* doc says 0x000A0601 */
```

Comment:
Update your PDF. The value was fixed some time ago (was a documentation issue). Meanwhile cap version 107 was released.

Reply:
Oh nice, I sent an email to the docs team about that.  Glad they fixed it.

Reply:
@pnowack I believe this is resolved.